packages/mongodb-dialects/java-driver/src/test/kotlin/com/mongodb/jbplugin/dialects/javadriver/IntegrationTest.kt

```diff
@@ -49,6 +51,12 @@ annotation class ParsingTest(
     @Language("java") val value: String,
 )
 
+@Retention(AnnotationRetention.RUNTIME)
+annotation class WithFile(
+    val fileName: String,
+    @Language("java") val value: String,
+)
```

Comment:
What does this do?

Reply:
Ahh I just read the dependent code, is it for keeping a reference so that it does not get garbage collected?

Reply:
It is actually to add more files to a project. Sometimes we need more than one file and I thought an annotation would be clear.